app/locales/en/translations.js

```diff
@@ -34,4 +41,61 @@ export default {
         sha2_description: 'SHA-2 is a cryptographic hash function designed by the NSA used to verify data integrity.',
         md5_description: 'MD5 is an algorithm used to verify data integrity.',
     },
+    dashboard: {
+        title: 'Dashboard',
+        create_new_project_button: 'Create new project',
+        quicksearch: {
+            search: 'Search your projects',
+            other_links: 'Go to <a href="/myprojects/">My Projects</a> to organize your work or <a href="/search/">search</a> the OSF',
```

Comment:
"my projects"
(though there's an argument we're using it like a proper noun, I think it looks especially wrong next to the "search" link)